target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -194,6 +194,10 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "1:wan" "2:lan:3" "3:lan:2"
 		;;
+	engenius,emr3000-v2)
+		ucidef_set_interface_wan "eth0"
+		ucidef_set_interface_lan "eth1"
```

Comment:
Just add your device to the big block above that sets "ucidef_set_interfaces_lan_wan "eth1" "eth0""

Reply:
Thanks for comments.